services/tenant-ui/frontend/src/components/reservation/Status.vue

```diff
@@ -102,6 +103,17 @@ const toast = useToast();
 const reservationStore = useReservationStore();
 const { loading, status } = storeToRefs(useReservationStore());
 
+const spinnerComputed = () => {
+  let icon = '';
+  if (loading.value) {
+    icon = 'pi pi-spin pi-spinner';
+  } else {
+    icon = 'pi pi-check';
+  }
+  console.log('icon', icon);
+  return icon;
+};
+
```

Comment:
Not needed, built into primevue buttons, see how other buttons in the app do it:
![image](https://user-images.githubusercontent.com/17445138/228347841-8aa033df-f882-43ef-b433-da9dc54ba697.png)